code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -280,6 +281,25 @@
 			step_sound = sound_list[pick(sound_list)]
 	..()
 
+/obj/item/clothing/shoes/clown_shoes/advanced/emag_act(var/mob/user) //Causes the shoes to play a sound every step instead of 2
+	..()
+	if(!emagged)
+		if(modulo_steps == 1)
+			to_chat(user, "<span class='warning'Aww shucks! Looks like someone beat you to the punch, for these shoes already play a sound every step! Perhaps it was the result of a very special day, or maybe it was tampered with by a very bored god. Either way, you are a fool!</span>")
+			to_chat(user, "<span class='good'>However, here is a consolation banana.</span>")
+			var/dat = {"<html><div><span style="color:#ff0000;">H</span><span style="color:#99ff00;">O</span><span style="color:#0000ff;">N</span><span style="color:#00ff80;">K</span><span style="color:#0066ff;">!</span></html></div>"}
+			to_chat(user, dat)
+			playsound(user, 'sound/items/bikehorn.ogg', 100, 0)
+			var/obj/item/weapon/reagent_containers/food/snacks/grown/banana/B = new()
+			B.name = "consolation banana"
+			B.desc = "This one seems to be enchanted..."
+			B.potency = 9001 //Honk
```

Comment:
This would cause players to slip 450 tiles (compared to 4 for a traitor peel or 10 for a cursed peel)

Reply:
feature

Reply:
I tested it before making the PR and it's intended, but as a consequence it also makes the player constantly move around while on the banana peel until the slipping is done, you just turn into a direction and it automatically moves you there.

Reply:
That means you can set the dramatic sound effect on the shoes, step on the peel and earrape everyone as an unstoppable doomed entity. Impossibly honked, barry redemption arc.